CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.8.1 (2020-11-10)
+
+- Fixed the country field test fixture to avoid exposing the DOM structure of underlying component. [#148](https://github.com/blackbaud/skyux-lookup/pull/148)
```

Comment:
```suggestion
- Fixed the country field test fixture to avoid exposing the DOM structure of the underlying component. [#148](https://github.com/blackbaud/skyux-lookup/pull/148)
```